platform/javascript/export/export_plugin.h

```diff
@@ -40,6 +40,7 @@
 #include "main/splash.gen.h"
 #include "platform/javascript/logo.gen.h"
 #include "platform/javascript/run_icon.gen.h"
+#include "core/os/time.h"
```

Comment:
You should include the header where you use it (in the .cpp). Also make sure to follow the style guidelines: https://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html#header-includes